### PR TITLE
Fix up examples, export actor_animation.dart

### PR DESCRIPTION
**Pull request by dnfield**
I'd like to make a change to `RenderObject` that breaks this repo (https://github.com/flutter/flutter/pull/83790). I have plans to submit a patch to fix it, but first this just cleans up the examples and adds a missing export for `ActorAnimation`.

A significant amount of changes here will be easier to review if you ignore whitespace changes :)

### CODE REVIEW

File: flare_flutter/lib/flare.dart [hunk 1]
```diff
@@ -31,10 +31,11 @@ import 'package:flare_flutter/trim_path.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 
-export 'package:flare_flutter/base/math/vec2d.dart';
-export 'package:flare_flutter/base/math/mat2d.dart';
-export 'package:flare_flutter/base/actor_node.dart';
 export 'package:flare_flutter/base/actor_artboard.dart';
+export 'package:flare_flutter/base/actor_node.dart';
+export 'package:flare_flutter/base/animation/actor_animation.dart';
```
Comment (dnfield) [inline flare_flutter/lib/flare.dart:36]: This is really the only critical line in this patch.

Comment (luigi-rosso) [reply flare_flutter/lib/flare.dart:36]: This looks totally fine, thanks for doing this!

**Review by luigi-rosso — APPROVED** [review]
LGTM thanks @dnfield

**Comment by luigi-rosso** [comment]
Generally available in ```3.0.1```!

**Pull request merged**